furniro-front-end/src/app/body/Body.tsx

```diff
@@ -0,0 +1,5 @@
+import Background from "./images/background.png";
+const Body = () => {
+  return;
+};
+export default Body;
```

Comment:
Do not use body as a name for page or component, it should be divided in sections. As to, if you are now going to make the banner for the home page, you should name it for what it is. And it should be in the module Home

Reply:
A module with the name Commons with the modules components and images inside was created, with the file HomeBanner.tsx inside the module components and a background.png image inside the module images.